src/Adapter/MySQL.php

```diff
@@ -306,6 +306,23 @@ protected function getFieldMapping()
         return $filterToTableMapping;
     }
 
+    /**
+     * Get the joined and escaped value from an multi-dimensional array
+     *
+     * @param string $separator
+     * @param array $values
+     *
+     * @return string Escaped string value
+     */
+    protected function getJoinedEscapedValue($separator, array $values)
+    {
+        return implode($separator, array_map(function ($value) use ($separator) {
```

Comment:
Using anonymous function is not the best for performance :thinking:

Reply:
@PierreRambaud Although I wanted to reply you with "Premature optimization is the root of all evil" 🤣, but you are right, anonymous functions in this case can increase up to two times the time required.

So, I have optimized the function to use `foreach` instead of using `array_map` with a callback function. This could reduce up to 40% of the time required.